src/main.ts

```diff
@@ -1,5 +1,11 @@
 import * as core from "@actions/core"
 import * as github from "@actions/github"
+import { GetResponseDataTypeFromEndpointMethod } from "@octokit/types"
+import { Octokit } from "@octokit/rest"
+import { WebhookPayload } from "@actions/github/lib/interfaces"
+
+const octokit = new Octokit()
+type GetPullResponseDataType = GetResponseDataTypeFromEndpointMethod<typeof octokit.pulls.get>
```

Comment:
I followed the process of generating the octokit [here](https://github.com/octokit/types.ts). Not sure if there's a cleaner way to do this.

Reply:
It looks good to me.

Reply:
Actually, it doesn't seem to compile. The fix is easy though, I think we can just drop this type since it's compatible with `WebhookPayload["pull_request"]`.